logging/log4j1_2/build.gradle

```diff
@@ -33,5 +33,5 @@ projectPomDescription = "This module provides a $project.msftAppInsights appende
 // endregion Publishing properties
 
 dependencies {
-    compile group: 'log4j', name: 'log4j', version: '1.2.17'
+    compile group: 'org.apache.logging.log4j', name: 'log4j-core', version: '2.15.0'
```

Comment:
this is specifically a log4j 1.2 appender, so need to still need to compile against 1.2.x, we can make it compileOnly similar to what we did for log4j2 appender in #2002

Reply:
Update.